addons/microdagr/stringtable.xml

```diff
@@ -497,6 +497,27 @@
             <Chinesesimp>无 (无法检视地图)</Chinesesimp>
             <Chinese>無 (無法檢視地圖)</Chinese>
         </Key>
+        <Key ID="STR_ACE_MicroDAGR_WaypointPrecision_DisplayName">
+            <English>Waypoint Precision</English>
+            <German>Wegpunkt Genauigkeit</German>
+        </Key>
+        <Key ID="STR_ACE_MicroDAGR_WaypointPrecision_Description">
+            <English>Controls how precise the waypointdistance can be displayed.</English>
```

Comment:
remove the punctuation at the end. Same for all the other new strings.

Reply:
Done. Can you show me how to make "code snippets" ?

Reply:
which one? You mean the code displayed above my comment? Yeah that's straight from the diff.